routing/src/config/interface.rs

```diff
@@ -112,7 +112,10 @@ impl ReconfigInterfacePlan {
         iftw: &mut IfTableWriter,
         vrftable: &VrfTable,
     ) -> Result<(), RouterError> {
-        self.enforce_deletions(iftw)?;
+        if false {
```

Comment:
comment to explain this in the code instead of just the commit?

Reply:
Ok!